contrib/buildahimage/Containerfile

```diff
@@ -1,40 +1,59 @@
-# git/Dockerfile
+# [stable|testing|upstream]/Containerfile
+#
+# Build a Buildah container image from the latest version
+# of Fedora.
+#
+# RELEASE defaults to stable if unset
+#
+# RELEASE=stable    aquires a stable version of Buildah
+#                   from the Fedoras Updates System.
+# RELEASE=testing   aquires a testing version of Buildah
+#                   from the Fedoras Updates System.
+# RELEASE=upstream  aquires a testing version of Buildah
+#                   from the Fedora Copr Buildsystem.
+#                   https://github.com/containers/buildah
+#
+# https://bodhi.fedoraproject.org/updates/?search=buildah
 #
-# Build a Buildah container image from the latest
-# upstream version of Buildah on GitHub.
-# https://github.com/containers/buildah
 # This image can be used to create a secured container
 # that runs safely with privileges within the container.
-# The containers created by this image also come with a
-# Buildah development environment in /root/buildah.
 #
+
 FROM registry.fedoraproject.org/fedora:latest
+ARG RELEASE="${RELEASE:-stable}"
```

Comment:
I'm slightly concerned the build-arg `RELEASE` is somewhat `ambiguous`.  It could be interpreted by a maintainer as meaning the buildah release version - which it most certainly is not.

I'm not so great at naming things...but would you consider something like "STAGE" or "FLAVOR" or similar?

Reply:
Yeah, I'm going to need to coordinate this with my build script changes.  Any objection to calling it "STAGE"?

Reply:
Stage means something in buildah.

Stick with FLAVOR, since nothing conflicts with that name.

Reply:
This line is equivalent to `ARG RELEASE=stable`.

Reply:
Good catch nalid! I keep doing that. It's one of the language context switch glitches my brain seems to hold onto, shell vs containerfile. At least I didn't try to use any javascript, rust, or c syntax! 🤣